google/cloud/forseti/common/email_templates/inventory_summary.jinja

```diff
@@ -19,6 +19,9 @@ limitations under the License.
 <body>
     Inventory Index Id: {{ inventory_index_id }}<br>
     Inventory Timestamp: {{ timestamp }}<br>
+    G Suite Domain-wide Delegation:
+    <a href="https://forsetisecurity.org/docs/latest/configure/inventory/gsuite.html">
```

Comment:
The hyperlink should surround `G Suite Domain-wide Delegation`, because there's more context of what the link is for.

Reply:
Changed!